world/dominion/plots/models.py

```diff
@@ -1064,6 +1064,14 @@ class PlotAction(AbstractAction):
         related_name="actions",
         on_delete=models.SET_NULL,
     )
+    episode = models.ForeignKey(
+        "character.Episode",
+        db_index=True,
```

Comment:
`db_index=True` is the default for foreign keys - default args should be omitted